info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -290,15 +288,18 @@ private boolean checkIncumbency(Map<String, String> map, String position, String
       String officeName;
       JsonArray officialIndicesArray;
       JsonArray officials;
+      List<String> incumbents;
```

Comment:
You seem to be spelling `incumbents` 3 different ways in this function.  Also, I don't think you need to have all of the declarations upfront for the various Jsonobjects/arrays/lists.  Probably fine to have them in the for loop if you're not using them beforehand.